src/qt/receivecoinsdialog.cpp

```diff
@@ -267,9 +267,18 @@ void ReceiveCoinsDialog::copyColumnToClipboard(int column)
 // context menu
 void ReceiveCoinsDialog::showMenu(const QPoint &point)
 {
-    if (!selectedRow().isValid()) {
+    QModelIndex sel = selectedRow();
+    if (!sel.isValid()) {
         return;
     }
+
+    // disable context menu actions when appropriate
+    const RecentRequestsTableModel * const submodel = model->getRecentRequestsTableModel();
+    const RecentRequestEntry *req = &submodel->entry(sel.row());
+    copyLabelAction->setEnabled(!req->recipient.label.isEmpty());
```

Comment:
nit: negations could be avoided (simplified) by using `setDisabled` instead (same for all three similar lines).

Reply:
addressed in 336a643